test/src/org/exist/xquery/TransformTest.java

```diff
@@ -49,8 +49,54 @@ public void transform() throws XMLDBException {
                 "<p>End Template 1</p>" +
                 "</doc>");
     }
-    
-    
+
+    @Test
+    public void isSameNodeTest() throws XMLDBException {
```

Comment:
Yeap, that test for [issue](https://github.com/eXist-db/exist/issues/1325). The problem at `isSameNode` method and that's why test have it name.